### DISCUSSION TITLE
Adding option to disable list update when an item is hovered over only on a draggable container

### DISCUSSION THREAD

**Discussion by alois-trancy** [formkit/drag-and-drop#65] [General]
Current behavior is like this.

https://github.com/formkit/drag-and-drop/assets/8169001/32cb8363-8dad-4994-9cf3-18cd8a47b6f5

As long as the item is hovered over a draggable container, the lists will be updated. Thus, leaving no possibility of cancelling the drag-and-drop action, aside from dragging and dropping it back to old container.

I hope there will be an option to only update the list when the item being dragged is dropped into a container.

**Comment by sashamilenkovic** [comment]
@alois-trancy  I'll get back to you in this discussion when this is implemented, whether via. plugin or part of a core option.

**Reply by sashamilenkovic** [reply]
@alois-trancy this is now a feature of the `dropOrSwap` or `insert` plugin: https://drag-and-drop.formkit.com/#drop-or-swap and is available on `v0.2.0` of @fomrkit/drag-and-drop
